core/src/main/java/io/javaoperatorsdk/jenvtest/binary/OSInfo.java

```diff
@@ -1,30 +1,37 @@
 package io.javaoperatorsdk.jenvtest.binary;
 
 public class OSInfo {
+  private final String os;
+  private final String arch;
 
-  public String getOSName() {
-    String os = System.getProperty("os.name").toLowerCase();
+  public OSInfo() {
+    final var osArch = System.getProperty("os.arch").toLowerCase();
+    switch (osArch) {
+      case "ppc64":
+        arch = "ppc64le";
+        break;
+      case "aarch64":
+        arch = "arm64";
```

Comment:
there are no binaries for ARM, but on macs this should work with amd64 binaries.

Reply:
There are binaries for ARM and, no, `amd64` won't work on more recent Macs. I've checked on both.

Reply:
ahh, sorry your are right, I completely missed that